lib/uklock/include/uk/spinlock.h

```diff
@@ -37,6 +37,10 @@
 extern "C" {
 #endif
 
+#include <uk/lock-common.h>
+#define _IRQFNAME(funcname) funcname##_irqf
```

Comment:
This should be removed, given it is already defined in the "lock-common" header.